app/_layouts/docs-v2.html

```diff
@@ -92,12 +92,13 @@
       <div class="page-content">
         <div class="content show-anchor-links">
           {% unless page.release.latest? or page.no_version == true %}
+            {% if page.release.label != nil %}
```

Comment:
This was an issue before, but I could create a label called “legacy” and it would show “unreleased”. Should we use the value for `label` in the banner message?

Reply:
Hmm, didn't even think of that as a potential problem. Let me test.